src/components/new-request/search.vue

```diff
@@ -271,6 +285,12 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin, Sear
   }
 
   get showCompanyTypeRadioButtons (): boolean {
+    if (
+      this.isConversion &&
+      !!this.getSearchBusiness &&
+      !!this.getConversionType
+    ) return true
```

Comment:
I think you can simply add the needed logic to line 98 and then we can eventually get rid of this getter.

Reply:
Clean! I love it!